commands/command_dedup.go

```diff
@@ -114,6 +114,10 @@ func dedup(p *lfs.WrappedPointer) (success bool, err error) {
 	srcFile := cfg.Filesystem().ObjectPathname(p.Oid)
 	dstFile := filepath.Join(cfg.LocalWorkingDir(), p.Name)
 
+	if srcFile == os.DevNull {
+		return true, nil
+	}
+
```

Comment:
I suppose this could move up to just below where `srcFile` is initialized, since we don't need `dstFile` yet.  But it's a minor thing.

Reply:
I've moved it.